src/component/category/AddEditCategory.jsx

```diff
@@ -104,7 +122,41 @@ const AddEditCategory = ({ id, categoryData, getDataTable, onClose }) => {
           />
         )}
       />{" "}
-      <Box sx={{ display: "flex", alignItems: "center", gap: "16px" }}>
+      <Controller
+        name="description"
+        control={control}
+        render={({ field }) => (
+          <TextField
+            {...field}
+            className={formStyles.formControl}
+            label="Description*"
+            variant="outlined"
+            fullWidth
+            multiline
+            margin="normal"
+            error={!!errors.description}
+            helperText={errors.description?.message}
+            size="small"
+            sx={{ mb: 2 }}
+            minRows={2}
+            maxRows={4}
+          />
+        )}
+      />{" "}
+      <Box
+        sx={{
+          display: "flex",
+          alignItems: "center",
+          justifyContent: "flex-end",
+          gap: "16px",
+        }}
+      >
+        <Button
+          type="button"
+          variant={"cancel"}
```

Comment:
```suggestion
          variant="cancel"
```
unwanted braces, fix at other places as well